src/components/common/card-course/CardCourse.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Card, Chip, Rating } from '@mui/material';
+import Link from 'next/link';
+
+import Video from '@/components/common/video';
+
+import styles from './CardCourse.module.scss';
+export interface CardCourseProps {
+  image: string;
+  courseId: string;
+  title: string;
+  tags: string;
+  rating: number;
+  description: string;
+  skills: string[];
+  previewVideo: string;
+  lessonsCount: number;
+}
```

Comment:
Similar interface declared in api/.../getCoursersDTO.ts. Instead making new interfaces, make general-purpose Course/Lesson interfaces, which will be used both in component props and DTO's.